hosters/hetzner-dedicated/hetzner-dedicated-wipe-and-install-nixos.sh

```diff
@@ -237,51 +320,76 @@ cat > /mnt/etc/nixos/configuration.nix <<EOF
   # We do not worry about plugging disks into the wrong machine because
   # we will never exchange disks between machines.
   environment.etc."mdadm.conf".text = ''
-    HOMEHOST hetzner
+    HOMEHOST $HOSTNAME
```

Comment:
Should we not keep the convention of always using `hetzner`? Otherwise changing the hostname could result in issues if I'm understanding this correctly

Reply:
This variable is only used in this script. Changing the hostname later won't change this particular file. The only important part is that you create the array with the same hostname you put in here.